db/seeds/development/pickup_posts.seeds.rb

```diff
@@ -0,0 +1,9 @@
+after "development:posts" do
+  site1 = Site.find_by!(name: "site1")
+
+  posts = site1.posts.limit(3)
+
+  posts.each_with_index do |post, index|
```

Comment:
You can give offset to `Enumerable#with_index`.
ref: https://docs.ruby-lang.org/ja/latest/method/Enumerator/i/with_index.html